waku-bindings/src/node/management.rs

```diff
@@ -1,18 +1,18 @@
 //! Node lifcycle [mangement](https://rfc.vac.dev/spec/36/#node-management) related methods
 
 // std
-use multiaddr::Multiaddr;
 use std::ffi::CString;
 // crates
-use libc::*;
+use libc::c_void;
+use multiaddr::Multiaddr;
 // internal
 use super::config::WakuNodeConfig;
-use crate::general::{PeerId, Result};
+use crate::general::Result;
 use crate::utils::{get_trampoline, handle_json_response, handle_no_response, handle_response};
 
 /// Instantiates a Waku node
 /// as per the [specification](https://rfc.vac.dev/spec/36/#extern-char-waku_newchar-jsonconfig)
-pub fn waku_new(config: Option<WakuNodeConfig>) -> Result<()> {
+pub fn waku_new(config: Option<WakuNodeConfig>) -> Result<*mut c_void> {
```

Comment:
Future PR will wrap this ptr right?

Reply:
Yes, i want to create an opaque struct to not deal with this raw pointer!